docs/machine-learning/resources/transforms.md

```diff
@@ -1,9 +1,10 @@
 ---
```

Comment:
Why are all the references here to the Legacy transforms?
<xref:Microsoft.ML.**Legacy**.Transforms.CategoricalOneHotVectorizer>

Reply:
out of scope for this docs release